tests/run_all_tests.sh

```diff
@@ -53,22 +53,27 @@ else
 fi
 
 # Per-example tests.
-if [[ $ALL_EXAMPLES == 'true' ]]; then
-  # we apply pytest within each example to avoid pytest's annoying test-filename collision.
-  # In pytest foo/bar/baz_test.py and baz/bleep/baz_test.py will collide and error out when
-  # /foo/bar and /baz/bleep aren't set up as packages.
-  for egd in $(find examples -maxdepth 1 -mindepth 1 -type d); do
-      pytest $egd
-  done
-fi
-
-# Per-example tests for Linen examples.
-for egd in $(find linen_examples -maxdepth 1 -mindepth 1 -type d); do
+#
+# we apply pytest within each example to avoid pytest's annoying test-filename collision.
+# In pytest foo/bar/baz_test.py and baz/bleep/baz_test.py will collide and error out when
+# /foo/bar and /baz/bleep aren't set up as packages.
+for egd in $(find examples -maxdepth 1 -mindepth 1 -type d); do
     pytest $egd
+
+    # Skip pytype on deprecated example
+    # TODO: Remove after github.com/google/flax/issues/567 is resolved
+    if [[ "$egd" == "examples/lm1b_deprecated" ]]; then
+        continue
+    fi
+
     # use cd to make sure pytpe cache lives in example dir and doesn't name clash
     # use *.py to avoid importing configs as a top-level import which leads tot import errors
     # because config files use relative imports (e.g. from config import ...). 
-    (cd $egd ; pytype "*.py")
+  if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
Maybe extract this into a function if we're using it twice?
Currently it prints the same message for every example.

Reply:
I restructured this code to run pytype separately, and will file a new PR because this one became stale.